### ISSUE TITLE
Update to Emscripten 2.x

### ISSUE THREAD

**Issue by JCash** [defold/defold#4897]
One outstanding issue with updating to 1.39.16 was that there was a bug in size minification for the js-web target.

This issue is now fixed in 1.39.17

https://emscripten.org/docs/introducing_emscripten/release_notes.html

**Comment by JCash** [comment]
Currently held up by a new issue:
https://github.com/emscripten-core/emscripten/issues/11273

**Comment by JCash** [comment]
Currently held up by a new issue:
https://github.com/emscripten-core/emscripten/issues/13038
